src/options.c

```diff
@@ -294,10 +296,20 @@ PRINCE_USAGE \
 JOHN_USAGE_REGEX \
 "--subsets[=CHARSET]        \"Subsets\" mode (see doc/SUBSETS)\n" \
 "--subsets-required=N       The N first characters of \"subsets\" charset are\n" \
-"                           the \"required set\"\n" \
 "--subsets-min-diff=N       Minimum unique characters in subset\n" \
 "--subsets-max-diff=[-]N    Maximum unique characters in subset (negative N is\n" \
 "                           relative to word length)\n" \
+"--inc2						\"inc2\" generation mode\n" \
```

Comment:
`inc2` means nothing. We told you this:
* You must introduce your proposal with some use for it;
* Maybe using a published paper.

Reply:
What it does :
1 Uses three lists of strings that are using position frequencies, the first is the normal one (no chain), the second is the chained characters, and the third are the complements of these.
The switch that you see inside the iteration on the word and the advance of the state maps to that.

2 Splits hardcoded strings by chuncks of 4, the order inside these chuncks being random.

3 Checks for the previous character on the fly, if it’s a special one (that can form a pattern like a « t » would often be followed by « h »), we switch set and get about 2 to 5 characters to proceed from the list « chainFreq », when they are done we pass on to the complement which come from the « counterChainFreq » and has been randomized inside « chrsts2 », when these are done we put the variable inc[pos-1] at zero, as if the placeholder was not following a special char (which we don’t know yet and will check back in word generation).

Why would we use it :
I don’t know it looks right on stdout, maybe the chunks should be of length 2 to increase rotation, or even hold just one char, but the smallest the chuncks get, the more there are of them, and it becomes pointless since chuncks index are proceeded from the end to the start like the states, and having chuncks of 1 for instance would cancel the splitting... Maybe a size of 3 is good.
The randomness is limited for each placeholder but them being put together then it gets very random, while still respecting frequency by position and character chains.

If you test it against other modes please limit them to lower since I have not implemented numbers nor alpha (you can see in the code that I did prepare some of the strings for it though).